.travis.yml

```diff
@@ -3,9 +3,9 @@ cache:
   directories:
   - deps
 elixir:
-  - 1.6.4
```

Comment:
I think we should keep support for older Elixir versions to remain backward compatible.

Reply:
@AndrewDryga :+1: . I Bumped it only because of the mix formatter discrepancy. That's not a big deal though. I'll revert that.